mitodl/ocw-www
# content only

#### What are the relevant tickets?
Related to https://github.com/mitodl/ocw-hugo-themes/pull/1

#### What's this PR do?
In the above PR, we migrated the styles, JS and templates from this repo over into `ocw-hugo-themes`.  This allows any repository of Hugo content to inherit the theme used to build `ocw-www` and OCW nextgen courses.  This PR removes everything aside from the content and a few boilerplate files that pull in the theme.

#### How should this be manually tested?
This has already been tested in the attached PR against this branch, and really this PR is part of that PR.  It should only be merged once the above PR and all related PR's preparing the release process to use `ocw-hugo-themes` have been deployed.  I will update this PR with references to the others as they become ready.
